pkg/local_object_storage/blobstor/fstree/range_bench_test.go

```diff
@@ -0,0 +1,89 @@
+package fstree_test
+
+import (
+	"testing"
+
+	"github.com/nspcc-dev/neofs-node/pkg/core/object"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/fstree"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/util/logicerr"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/stretchr/testify/require"
+)
+
+func oldRange(fsTree *fstree.FSTree, addr oid.Address, from, length uint64) ([]byte, error) {
```

Comment:
looks weird, not usually done like that. Instead, precede the optimization with a commit adding benchmark test. Then goes optimization commit carrying comparison result (`benchstat` is good for this)